manager/manager.go

```diff
@@ -764,6 +764,8 @@ func (m *Manager) rotateRootCAKEK(ctx context.Context, clusterID string) error {
 			return err
 		}
 	} else {
+		log.G(ctx).Warn("Encrypting the root CA key in swarm using environment variables is deprecated. " +
+			"Support for decrypting or rotating the key will be removed in the future.")
```

Comment:
Let's move this up to just after `if strPassphrase == "" {` so we can warn if someone tries to encrypt a root key that wasn't already encrypted.